CHANGELOG.md

```diff
@@ -8,9 +8,17 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ## [next-version]
 
+## Added
+
+## Changed
+
+- UI: Update Face Liveness Video Challenge screen UI
+
+## Fixed
+
 ## [6.8.0] - 2021-05-13
 
-## Added
+### Added
```

Comment:
Curious, if this change why not the above ones?

Reply:
Fixed the heading levels for those now.